<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowContents</class>
 <widget class="QWidget" name="MainWindowContents">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="cloudGroup">
     <property name="title">
      <string>Cloud Browser</string>
     </property>
     <layout class="QFormLayout" name="cloudFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="remoteLabel">
        <property name="text">
         <string>Cloud remote</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="remoteRowLayout">
        <item>
         <widget class="QComboBox" name="remoteCombo"/>
        </item>
        <item>
         <widget class="QPushButton" name="refreshRemotesButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="folderLabel">
        <property name="text">
         <string>Top-level folder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="folderCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="folderTree">
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
